core/src/main/java/org/apache/iceberg/util/SerdeUtil.java

```diff
@@ -28,14 +28,14 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Base64;
 
-public class SerializationUtil {
+public class SerdeUtil {
 
-  private SerializationUtil() {
+  private SerdeUtil() {
```

Comment:
Are these change related? Why the rename?

Reply:
Not realized there is a `SerializationUtil` in `mr` module at begin, so create a new `SerdeUtil` in `core`. I will revert the name changed.